src/xinterpreter_wasm.cpp

```diff
@@ -17,7 +17,13 @@ namespace xcpp
 {
 
     wasm_interpreter::wasm_interpreter()
-        : interpreter(0, nullptr)
+        : interpreter(create_args().size(), create_args().data())
```

Comment:
```suggestion
        : interpreter({"-std=c++20"})
```

Reply:
Hmm not sure we can support as this as per how the interpreter class is defined.

https://github.com/compiler-research/xeus-cpp/blob/a7649c0e901f4618a6ce60df2155716959346288/include/xeus-cpp/xinterpreter.hpp#L33-L38

Hence the probably the best way to do this is to using something to create the arguments ?

Reply:
In that case you can set this up in `createInterpreter` in case of Emscripten.

Reply:
Yeah that something I was going for earlier.